versions.json

```diff
@@ -4,6 +4,7 @@
         "version_string": "1.11.0",
         "license": "Apache License 2.0",
         "consortium_build_number": "2",
+        "package_revision": "3",
```

Comment:
In the PR description, you mentioned 2 for this. For some of them, it's now 3. Why?